merino/middleware/geolocation.py

```diff
@@ -39,6 +41,7 @@ class Location(BaseModel):
     postal_code: Optional[str] = None
     key: Optional[str] = None
     coordinates: Optional[Coordinates] = None
+    city_names: Optional[dict[str, str]] = None
```

Comment:
`record.city.names` is of type `dict[str, str]`.
```suggestion
    city_names: dict[str, str] = {}
```